src/rules/javascript/no_document_write.js

```diff
@@ -0,0 +1,29 @@
+import { NO_DOCUMENT_WRITE } from 'messages/javascript';
+import { getNodeReferenceName } from 'utils';
+
+export default function(context) {
+  return {
+    CallExpression: function(node) {
+      var objectName;
+      if (typeof node.callee.object !== 'undefined' &&
+          node.callee.object.type === 'Identifier') {
+
+        objectName = getNodeReferenceName(context, node.callee.object);
```

Comment:
Can you comment bits like this - just to make it clearer what's going on? Just helps with immediacy later on.